templates/cemanager.yaml

```diff
@@ -136,26 +136,18 @@ spec:
                 - curl
                 - -s
                 - localhost:8080/cem-healthz
-            initialDelaySeconds: 30
-            failureThreshold: 12
-            # Poll the manager every 20 seconds. If we fail 12 times (4
-            # minutes) the manager is most likely in a bad spot. It's possible
-            # for migrations to take extended periods of time, so we need to
-            # account for that.
-            periodSeconds: 20
+            initialDelaySeconds: 10
```

Comment:
do we even need this now? startup should be roughly instant right?

Reply:
For some reason I still see pods fail to connect to pg for a few seconds as it first starts up, even if pg is available the entire time.
```
$ klo -n coder cemanager-98674fc85-mrc52 cemanager
Waiting on db
172.28.48.3:5432 - no response
172.28.48.3:5432 - no response
172.28.48.3:5432 - accepting connections
...
```
I'm guessing its some Istio thing.